var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -68,29 +69,35 @@ class Petsc(Package):
 
     depends_on('hdf5+mpi', when='+hdf5+mpi')
     depends_on('parmetis', when='+metis+mpi')
-    # Hypre does not support complex numbers.
-    # Also PETSc prefer to build it without internal superlu, likely due to conflict in headers
-    # see https://bitbucket.org/petsc/petsc/src/90564b43f6b05485163c147b464b5d6d28cde3ef/config/BuildSystem/config/packages/hypre.py
+    # Hypre does not support complex numbers.  Also PETSc prefer to build it
+    # without internal superlu, likely due to conflict in headers see
+    # https://bitbucket.org/petsc/petsc/src\
```

Comment:
maybe add `    # NOQA: ignore=E501` so that the link is not broken and it is easy to copy-paste it to a browser?